<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapWindow</class>
 <widget class="QMainWindow" name="MapWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MapWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>10</y>
      <width>391</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Legend</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSearch">
     <property name="title">
      <string>Search</string>
     </property>
     <widget class="QMenu" name="menuAvailable_Beds">
      <property name="title">
       <string>Available Beds</string>
      </property>
      <addaction name="actionAcute_Care"/>
      <addaction name="separator"/>
      <addaction name="actionComplex_Continuing_Care"/>
      <addaction name="separator"/>
      <addaction name="actionLong_Term_Care"/>
     </widget>
     <addaction name="menuAvailable_Beds"/>
    </widget>
    <addaction name="menuSearch"/>
    <addaction name="separator"/>
    <addaction name="actionLogout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuFacility">
     <property name="title">
      <string>Facility</string>
     </property>
     <addaction name="actionHospital"/>
     <addaction name="separator"/>
     <addaction name="actionNursing_Home"/>
     <addaction name="separator"/>
     <addaction name="actionView_All"/>
    </widget>
    <addaction name="menuFacility"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionFacilityView"/>
   </widget>
   <widget class="QMenu" name="menuAdmin">
    <property name="title">
     <string>Admin</string>
    </property>
    <addaction name="actionAdd_newUser"/>
    <addaction name="separator"/>
    <addaction name="actionRemove_User"/>
   </widget>
   <widget class="QMenu" name="menuReport">
    <property name="title">
     <string>Report</string>
    </property>
    <addaction name="actionGenerate"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuAdmin"/>
   <addaction name="menuReport"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHospital">
   <property name="text">
    <string>Hospital</string>
   </property>
  </action>
  <action name="actionNursing_Home">
   <property name="text">
    <string>Nursing Home</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="text">
    <string>Full Screen</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionOn">
   <property name="text">
    <string>On</string>
   </property>
  </action>
  <action name="actionOff">
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionAcute_Care">
   <property name="text">
    <string>Acute Care</string>
   </property>
  </action>
  <action name="actionComplex_Continuing_Care">
   <property name="text">
    <string>Complex Continuing Care</string>
   </property>
  </action>
  <action name="actionLong_Term_Care">
   <property name="text">
    <string>Long Term Care</string>
   </property>
  </action>
  <action name="actionAdd_newUser">
   <property name="text">
    <string>Create User</string>
   </property>
  </action>
  <action name="actionDelete_User">
   <property name="text">
    <string>Delete User</string>
   </property>
  </action>
  <action name="actionCreate_Facility">
   <property name="text">
    <string>Create Facility</string>
   </property>
  </action>
  <action name="actionDelete_Facility">
   <property name="text">
    <string>Delete Facility</string>
   </property>
  </action>
  <action name="actionFacilityView">
   <property name="text">
    <string>FacilityView</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="text">
    <string>Generate</string>
   </property>
  </action>
  <action name="actionView_All">
   <property name="text">
    <string>View All</string>
   </property>
  </action>
  <action name="actionRemove_User">
   <property name="text">
    <string>Remove User</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
